internal/db/schema/migrations/oss/postgres/78/01_storage_policies.up.sql

```diff
@@ -0,0 +1,160 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+create table policy (
+  public_id wt_public_id primary key,
+  scope_id wt_scope_id not null
+);
+comment on table policy is
+'policy contains entries that represent an abstract policy';
+
+create table storage_policy (
```

Comment:
```suggestion
create table policy_storage_policy (
```
Storage policy is in the `policy` package since it is a type of policy.